situp-plugins/elasticsearch/src/test/java/com/amazon/situp/plugins/sink/elasticsearch/ElasticsearchSinkIT.java

```diff
@@ -276,4 +328,111 @@ private boolean deleteDirectory(final File directoryToBeDeleted) {
     }
     return directoryToBeDeleted.delete();
   }
+
+  public static boolean isODFE() {
+    final boolean isODFE = Optional.ofNullable(System.getProperty("odfe"))
```

Comment:
variable name, this almost makes it seem like a recursive call because its the same name as the method